armi/runLog.py

```diff
@@ -503,9 +515,41 @@ class RunLogger(logging.Logger):
         :id: I_ARMI_LOG
         :implements: R_ARMI_LOG
 
+        This logger makes it easy for users to add log statements to
+        and ARMI application, and ARMI will control the flow of
+        those log statements. In particular, ARMI overrides the normal
+        Python logging tooling, to allow developers to pipe their log
+        statements to both screen and file. This works for stdout and
+        stderr.
+
+        The log statements themselves can be any text a user wants to
+        record during a run. For instance, basic notifications of what
+        is happening in the run, or basic warnings about something that
+        could be an issue, so you want people to know about. Likewise,
+        full errors can be recorded with this tool. These different
+        types of log statements are easily identified by specifying the
+        "verbosity" of the log statement, in the code where you do the
+        logging. The most important verbosities are: error, warning,
+        info (which is just to add information to the log), and debug
+        (which is meant to be extremely detailed and not always
+        desired).
+
+        And at the end of the ARMI-based simulation, the analyst doing
+        the simulation will have a full record of potentially intersting
+        information they can use to understand their run.
+
     .. impl:: Logging is done to the screen and to file.
         :id: I_ARMI_LOG_IO
         :implements: R_ARMI_LOG_IO
+
+        By default, this logger prints all log messages during an ARMI
+        simulation to screen and to a plain text file. That is, at any
+        place in the ARMI application, developers can interject a plain
+        text logging message, and when that code is hit during an ARMI
+        simulation, the text will be piped to screen and to a log file.
+        This helpful for analysts who want to watch their simulation in
+        real time, and for everyone that want a record of anything
+        interesting that happened during the run.
```

Comment:
I'd say this could get added to the basis or to additional info in the requirement, but isn't needed for the implementation. But it's fine if it stays too...I won't enforce this comment!

Reply:
Hmmm... That's an interesting point.

Reply:
Removed